packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -183,7 +183,7 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
             id={ this._id + item.key }
             className={ className }
             onClick={ (ev) => this._onItemClick(ev, item) }
-            data-command-key={ index }
+            data-command-key={ item.key }
```

Comment:
how about using the itemKey from line 169. This will allow item.key to be passed in with a default of index if no key is given.

Not sure if no key is a valid scenario, but at least we have less chance of breaking someone's usage already.

Reply:
Fixed.